airbyte-webapp/src/packages/cloud/views/auth/components/PersonQuoteCover/PersonQuoteCover.module.scss

```diff
@@ -0,0 +1,84 @@
+@use "../../../../../../scss/colors";
+@use "../../../../../../scss/fonts";
+
+.image {
+  position: relative;
+  height: 100vh;
+  background-image: url(../../../../../../../public/person-photo.png);
```

Comment:
The public folder is a bit unorganized. My suggestion is to put the photo under `/public/images/testimonials/cartdotcom-person-photo` and `/public/images/testimonials/cartdotcom-logo`

Reply:
Ok, sure, will update the location